database/migrations/2021_05_11_094857_create_addresses_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAddressesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('addresses', function (Blueprint $table) {
+            $table->id();
+            $table->string('street')->nullable();
+            $table->integer('buildingNO')->nullable();
```

Comment:
Use string instead of integer to be able to handle edge cases.
Ex: A building with number containing letters and numbers